### PR TITLE
feat: add 'accountUpdateDefaultWalletId' mutation

### PR DESCRIPTION
**Pull request by vindard**

### CODE REVIEW

File: src/graphql/root/mutation/account-update-default-wallet-id.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    const result = await Accounts.updateDefaultWalletId({
+      walletId,
+      accountId: domainAccount.id,
+    })
+
+    if (result instanceof Error) {
+      const appErr = mapError(result)
+      return { errors: [{ message: appErr.message || appErr.name }] }
```
Comment (dolcalmi) [inline src/graphql/root/mutation/account-update-default-wallet-id.ts:34]:
```suggestion
      return { errors: [mapError(result)] }
```

**Review by dolcalmi — APPROVED** [review]
just a minor comment.. feel free to merge

File: src/graphql/root/mutation/account-update-default-wallet-id.ts (hunk 1 above)
Comment (dolcalmi) [reply src/graphql/root/mutation/account-update-default-wallet-id.ts:34]: mapError already do message/name coalescing
```js
const errorName = error.name as ApplicationErrorKey
let message = error.message || errorName || ""
```

Comment (vindard) [reply src/graphql/root/mutation/account-update-default-wallet-id.ts:34]: With the current destructuring I get:
```
{
    "data": {
        "accountUpdateDefaultWalletId": {
            "errors": [
                {
                    "message": "Unknown error occurred (code: InvalidWalletId)"
                }
            ],
            "account": null
        }
    }
}
```

And with the direct `mapError(result)` suggested I get:
```
{
    "errors": [
        {
            "message": "Unknown error occurred (code: InvalidWalletId)",
            "code": "UNKNOWN_CLIENT_ERROR"
        }
    ],
    "data": null
}
```

**Pull request merged**